README.md

```diff
@@ -113,6 +113,16 @@ Perhaps Goby should be far easier for Rubyists to comprehend. You can use Ruby's
 - Flow control
     - `if`, `else`, `elsif`
     - `while`
+- Literals
+    - Numeric literal
+        - Integer: `100`, `-299`, `-0`
+        - Float: `3.14`, `-273.15`
```

Comment:
We currently can't parsing negative float like this, we need to wrap them like `-(273.15)`. see #526